### DISCUSSION TITLE
Xform no output

### DISCUSSION THREAD

**Discussion by hclaus** [LBNL-ETA/pyradiance#42] [General]
I am totally new to radiance, I try to follow Radiance_cookbook, but I am stuck in 2.1.2
cat lights.rad !xform -t 1 1 2.95 -a 3 -t 1 0 0 -a 4 -t 0 1 0 erco.rad
in python I don't have a lights.rad and want to add it (later) to B1.rad
tr=pr.Xform(inp="B1.rad").translate(1,1,2.95)
out_str = tr.args
print(out_str)
tr is empty and doesn't recognize decode().
How do I use Xform correctly?

**Comment by taoning** [comment]
Do this instead:
`tr=pr.Xform(inp="B1.rad").translate(1,1,2.95)()`
